apps/desks.py

```diff
@@ -561,20 +561,49 @@ def _find_other_sluglines(self, family_id, slugline, versioncreated, desk_id):
 
 class OverviewResource(Resource):
     url = r'desks/<regex("([a-f0-9]{24})|all"):desk_id>/overview/<regex("stages|assignments|users"):agg_type>'
+    privileges = {'POST': 'desks'}
     resource_title = "desk_overview"
-    resource_methods = ['GET']
+    resource_methods = ['GET', 'POST']
+    schema = {
```

Comment:
I think that it's a good time to introduce `flake8-quotes` plugin into the core.
The current schema has a single quote style, but the code below is replacing all single quotes to double quotes 😄